packit/api.py

```diff
@@ -1109,7 +1109,7 @@ def sync_from_downstream(
         no_pr: bool = False,
         fork: bool = True,
         remote_name: Optional[str] = None,
-        exclude_files: Iterable[str] = None,
+        exclude_files: Optional[Iterable[str]] = None,
```

Comment:
`Optional[Iterable[T]]` is an anti-pattern, though I have a feeling it will be very hard to eradicate…

Reply:
https://docs.astral.sh/ruff/rules/implicit-optional/

Is there anything I can change to make you happy? :)

Reply:
I'll probably open a follow-up for this, we have it almost everywhere :/